lib/internal/inspector_node_domains.js

```diff
@@ -0,0 +1,433 @@
+'use strict';
+
+const assert = require('assert');
+const EventEmitter = require('events');
+const util = require('util');
+
+const { registerDispatcherFactory } = process.binding('inspector');
+
+const domainHandlerClasses = new Map();
+const SessionTerminatedSymbol = Symbol('DisposeAgent');
+
+function createNotificationChannel(session, domain) {
+  return function(method, params) {
+    session.sendMessageToFrontend(JSON.stringify({
+      method: `${domain}.${method}`,
+      params
+    }));
+  };
+}
+
+function createResponseCallback(session, id) {
+  let first_call = true;
```

Comment:
For JS we usually go with camelCase, i.e. `firstCall`

Reply:
Done.